Firestore/Example/Tests/SpecTests/FSTSpecTests.mm

```diff
@@ -80,12 +92,21 @@ @implementation FSTSpecTests
 }
 
 - (BOOL)shouldRunWithTags:(NSArray<NSString *> *)tags {
-  @throw FSTAbstractMethodException();  // NOLINT
+  if ([tags indexOfObject:kNoIOSTag] != NSNotFound) {
+    return NO;
```

Comment:
This is like a quadruple negative. (So we search for the 'no' tag, and if the result is 'not' 'not' found, then we return 'no'.) It took me way more time than I'd like to admit to parse this.

I liked the version you added in leveldbspectests better, i.e.:
```
  if ([tags containsObject:kNoLRUTag]) {
    return NO;
  }
```

So maybe change this to:

```
if ([tags containsObject:kNoIOSTag]) {
  return NO;
}
```

Similarly for the other tags below.


A different alternative would be to write this in a style that more closely matches the android version. So have a set of disabled tags, then loop through the given tags and see if any are in the set of disabled tags.

Reply:
Thanks for the suggestion. I went with `containsObject` since that simplifies the logic by quite a bit without creating to much divergence.